core/controllers/collection_editor.py

```diff
@@ -155,6 +140,25 @@ def _require_valid_version(self, version_from_payload, collection_version):
                 % (collection_version, version_from_payload))
 
     @require_editor
+    def get(self, collection_id):
+        """Populates the data on the individual collection page."""
+
+        try:
+            # Try to retrieve collection
+            collection_dict = (
+                summary_services.get_learner_collection_dict_by_id(
+                    collection_id, self.user_id,
+                    allow_invalid_explorations=True))
+        except Exception as e:
+            raise self.PageNotFoundException(e)
+
+        self.values.update({
+            'collection': collection_dict
```

Comment:
I was following the same syntax as in collection_viewer.py from the CollectionDataHandler. Should we have them uniformly return the same attributes? Or should i leave the EditableCollectionDataHandler to return in a slightly different format? In Collection_viewer.py we have
`self.values.update({
            'can_edit': (
                self.user_id and rights_manager.Actor(self.user_id).can_edit(
                    feconf.ACTIVITY_TYPE_COLLECTION, collection_id)),
            'collection': collection_dict,
            'is_logged_in': bool(self.user_id),
            'session_id': utils.generate_new_session_id(),
        })`